assets/css/solcast.webflow.style.css

```diff
@@ -727,4 +731,46 @@ table tbody tr:last-child td:last-child{
 .gallery-items .item .item-wrap {
   background-color: #111;
   color: #acafb6;
-}
\ No newline at end of file
+}
+
+/* Rich Text Formatting */
+
+.section-banner-rich-text > h1, .section-banner-rich-text > h2, .section-banner-rich-text > h3 {
+	font-family: var(--font-families--headings) !important;
+  font-weight: 700 !important;
+  color: var(--text-color--text-alternate) !important;
```

Comment:
Not keen on so much !important

[When everything is !important, nothing is.](https://www.tacobunbun.com/blog/post/beef-up-your-web-design-skills-avoid-important-at-all-costs)

Reply:
yeah agreed, i think its better to user a more specific selector i.e. put an id on the parent like #section-banner > h1, ...